example/default.css

```diff
@@ -0,0 +1 @@
+../themes/default/default.css
```

Comment:
It's a symlink, this is how Github displays them I guess.

In a next iteration we'll really add the file from the build.